### DISCUSSION TITLE
b站资源占用问题

### DISCUSSION THREAD

**Discussion by PhoenixPtt** [aidenlx/media-extended#311] [Q&A]
播放前的CPU占用情况：
![image](https://github.com/PKM-er/media-extended/assets/68508116/f7e66f66-cdc8-4835-bb7d-f9d9a309dab5)

在obsidian中播放时CPU占用情况：
![image](https://github.com/PKM-er/media-extended/assets/68508116/8eb6b4ca-0adc-4229-88ee-d0458d9d6b87)

右键点击在默认浏览器中播放时CPU占用情况：
![image](https://github.com/PKM-er/media-extended/assets/68508116/82b3fbe2-cae6-4847-8b1b-329c2d115f8a)

在obsidian中播放时的CPU占用明显高于在浏览器中的资源占用。
麻烦研究下是否可以有节约资源占用的方法？

**Comment by aidenlx** [comment]
不是你播放啥网址文件之类的也发出来啊😂你这我怎么调试

最好能建个单独的库，把占用高的媒体放在里头发上来

**Comment by aidenlx** [comment]
假如是本地的测试下用media插件和原生的占用有没差很多

**Comment by PhoenixPtt** [comment]
不好意思，播放的是bilibili网站的视频，不是本地文件。bilibili上的视频，所有都是这个问题。

**Comment by aidenlx** [comment]
先试下切换编码有没帮助

https://github.com/PKM-er/media-extended/assets/31102694/fbdfc226-62c2-4d84-a435-1790bdc53d45

**Comment by bcdax110** [comment]
确实很有可能是视频编码的原因，很早之前就发现我这个机器（i7 7700HQ, GTX1060）在使用AV1编码（默认编码大概率也会自动选择AV1）的时候CPU占用就会直接爆炸，如果再进行二倍速的话，基本上点一下鼠标都得在一分钟之后才会有反应。用AVC编码对CPU消耗就明显比AV1低。估计是使用的CPU还不支持硬件解码AV1。

**Comment by aidenlx** [comment]
b站现在默认上av1就很恶心，chrome的av1编码支持是支持了，但是现在就没几台电脑有av1硬件解码的，其他都是软解，肯定CPU占用爆表

我怀疑浏览器占用正常可能是chrome用不支持av1的老版所以不会有这个问题😂
